test/test_fcntl.rs

```diff
@@ -197,22 +209,87 @@ mod linux_android {
         let mut buf = [0u8; 200];
         assert_eq!(100, read(fd, &mut buf).unwrap());
     }
+
+    #[test]
+    #[cfg(not(any(target_arch = "aarch64",
+                  target_arch = "arm",
+                  target_arch = "armv7",
+                  target_arch = "i686",
```

Comment:
Try "x86" instead of "i686".  Also, please explain in a comment why the test is disabled  on certain architectures.

Reply:
I've added a comment and switched to x86.